pulley/src/lib.rs

```diff
@@ -1278,6 +1278,39 @@ macro_rules! for_each_extended_op {
             vfma32x4 = Vfma32x4 { dst: VReg, a: VReg, b: VReg, c: VReg };
             /// `dst = ieee_fma(a, b, c)`
             vfma64x2 = Vfma64x2 { dst: VReg, a: VReg, b: VReg, c: VReg };
+
+            /// `dst_lo:dst_hi = lhs_lo:lhs_hi + rhs_lo:rhs_hi`
```

Comment:
maybe a style thing, or Verilog influence, or something, but usually I write `a:b` concat syntax MSB-first (as the concrete values would be written) -- so `dst_hi:dst_lo = ...` maybe?

Reply:
It's ok I have no influence so I think I swap lo:hi every time I write it, happy to canonicalize to hi:lo!